spec/rubocop/cop/style/extra_spacing_spec.rb

```diff
@@ -49,6 +49,28 @@
       expect(cop.offenses).to be_empty
     end
 
+    it 'does not skip single line hash literals with symbol keys' do
+      inspect_source(cop, 'hash = {a:   1,  b:    2}')
+      expect(cop.offenses.size).to eq(3)
+    end
+
+    it 'does not skip single line hash literals with hash rockets' do
+      source = [
+        'let(:single_line_hash) {',
+        '  {"a"   => "1", "b" => "2"}',
+        '}',
+        'let(:multiline_hash) {',
```

Comment:
Maybe not, I just wanted an example that had both single-line and multi-line hashes in the source.